docs/community/chat-and-discussion-forums.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Chat & discussion forums
+description: Connect with the Filecoin community in dicussion forums or on IRC
+---
+# Chat & Discussion Forums
+
+The Filecoin community is active and here to answer your questions in your channel of choice. Please use Slack or IRC for X. For longer-lived discussions and support, our Discussion Forums are the better choice.
+
+## Discussion Forums
+
+For long-lived discussions and for support, please use the discussion forums at [https://discuss.filecoin.io/](https://discuss.filecoin.io/) instead of IRC or Slack! It’s easy for complex discussions to get lost in a sea of new messages on IRC, and posting longer discussions and support requests on the forums helps future visitors, too.
```

Comment:
```suggestion
For long-lived discussions and for support, please use the discussion forums at [https://discuss.filecoin.io/](https://discuss.filecoin.io/) instead of Slack! It’s easy for complex discussions to get lost in a sea of new messages, and posting longer discussions and support requests on the forums helps future visitors, too.
```

Reply:
Again confused about why IRC references are being removed if you're recommending Matrix on the website (which I think is IRC ¯\_(ツ)_/¯ ).